docs/elements.md

```diff
@@ -1354,7 +1354,7 @@ Display a statically or dynamically generated image. Supports many image file fo
 | `file-name` | string  | —                       | Name of image file.                                                                                                                                                                                     |
 | `type`      | text    | `"static"`              | Type of file, either 'static' (an existing file) or 'dynamic' (a file generated by element or server code).                                                                                             |
 | `directory` | text    | `"clientFilesQuestion"` | The directory that contains the file, either `"clientFilesQuestion"` or `clientFilesCourse` (see [client and server files](clientServerFiles.md)). A directory cannot be specified if `type="dynamic"`. |
-| `width`     | number  | —                       | Width of image (e.g., '250px').                                                                                                                                                                         |
+| `width`     | number  | —                       | Width of image (e.g., '250').                                                                                                                                                                           |
```

Comment:
There are mistakes littered where we use `width: XXpx`, which shouldn't be valid HTML. I assume this is worth breaking out to a separate PR.

Reply:
Broken out into #11840